content/applications/hr/recruitment/refuse_applicant.rst

```diff
@@ -0,0 +1,137 @@
+=================
+Refuse applicants
+=================
+
+At any point in the recruitment process, an applicant can be refused for a job position. To refuse
+an applicant, click on an applicant's card to navigate to a detailed view of the card. At the top of
+the applicant's card, there are several buttons. Click the one labeled :guilabel:`Refuse`.
+
+.. _recruitment/refuse-reasons:
+
+Refuse reasons
+==============
+
+A :guilabel:`Refuse Reason` pop-up window appears. Click the desired :guilabel:`Refuse Reason` from
+the presented options. The default refuse reasons in Odoo, and their corresponding email templates,
+are:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Email Template
+     - Refusal Reason
+   * - :guilabel:`Recruitment: Refuse`
+     - | :guilabel:`Doesn't fit the job requirements`
+       | :guilabel:`Language issues`
+       | :guilabel:`Role already fulfilled`
+       | :guilabel:`Duplicate`
+       | :guilabel:`Spam`
+   * - :guilabel:`Recruitment: Not interested anymore`
+     - | :guilabel:`Refused by Applicant: don't like job`
+       | :guilabel:`Refused by Applicant: better offer`
+       | :guilabel:`Refused by Applicant: salary`
+
+Additional refusal reasons :ref:`can be created and existing ones can be modified or deleted
+<recruitment/new-refuse>`.
+
+.. note::
+   Pre-configured email templates in Odoo sometimes use dynamic placeholders, which are customized
+   pieces of data that populate dynamic content. For example, if the applicant's name is a piece of
+   dynamic content, the applicant's name appears anytime that dynamic placeholder appears on the
+   email template. For more detailed information on email templates, refer to the
+   :doc:`../../general/companies/email_template` documentation.
+
```

Comment:
To complete the basic flow in this topmost section, I'm suggesting to bring the action to click the refuse button all the way up here, with the image. 

that way, readers who are just coming to the doc to see how applicants are refused can see the workflow right from the beginning, and additional features, such as configuring refuse reasons and the refusal email template, are found in later sections.

But, this is just a suggestion! Feel free to just ignore this comment, or reach out to me and we can have a discussion about it (:

```suggestion

After selecting a refuse reason, click the :guilabel:`Refuse` button to close the application.

.. image:: refuse_applicant/refuse.png
   :align: center
   :alt: An applicant's card with the refused banner appearing in the top-right corner in red.
```